hasura/schema/admin/schema.graphql

```diff
@@ -33940,6 +33949,7 @@ type profiles {
     """
     where: vaults_bool_exp
   ): vaults_aggregate!
+  vec_similarity: float8
```

Comment:
@CryptoGraffe this is bad - not supposed to be here. Old migration causing it. Manually drop column `vec_similarity`